x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/RoleDescriptor.java

```diff
@@ -1165,13 +1408,15 @@ public interface Fields {
         ParseField GLOBAL = new ParseField("global");
         ParseField INDEX = new ParseField("index");
         ParseField INDICES = new ParseField("indices");
+        ParseField REMOTE_INDICES = new ParseField("remote_indices");
         ParseField APPLICATIONS = new ParseField("applications");
         ParseField RUN_AS = new ParseField("run_as");
         ParseField NAMES = new ParseField("names");
         ParseField ALLOW_RESTRICTED_INDICES = new ParseField("allow_restricted_indices");
         ParseField RESOURCES = new ParseField("resources");
         ParseField QUERY = new ParseField("query");
         ParseField PRIVILEGES = new ParseField("privileges");
+        ParseField REMOTE_CLUSTERS = new ParseField("clusters");
```

Comment:
While `clusters` makes sense at the API level, within the codebase it feels quite ambiguous, esp. since the already have `CLUSTER` so would prefer to be more explicit with `REMOTE_CLUSTERS` here.